DebuggerGame/Assets/Scripts/BoardObject Scripts/Player.cs

```diff
@@ -4,8 +4,6 @@
 
 public class Player : BoardObject
 {
-    public InventorySystem collection;
```

Comment:
There is still a collection reference on line 97 in this file

Reply:
Removed collection reference